gammapy/datasets/make.py

```diff
@@ -566,11 +611,16 @@ def make_test_eventlist(observation_table,
     # positive defined, so it is necessary to translate the (0, 1)
     # interval of the random variable to (emax, e_min) in order to
     # have a decreasing power-law
-    index = spectral_index + 1
     e_min = Quantity(0.1, 'TeV')
     e_max = Quantity(100., 'TeV')
-    energy = Quantity((e_min.value - e_max.value) *
-                      random_state.power(a=index, size=n_events) + e_max.value, 'TeV')
+    energy = Quantity(sample_powerlaw(e_min.value, e_max.value,
+                                      spectral_index + 1, # TODO: need to add 1
+                                                          # to the index because
+                                                          # sample_powerlaw
+                                                          # subtracts it, why??!!!
+                                      size=n_events,
+                                      random_state=random_state),
+                      'TeV')
```

Comment:
I'd say bug.

See http://nbviewer.ipython.org/gist/cdeil/3605f1af22d51d0060bb for a check and recommendation what to do (change index by 1 and improve docstring of `sample_powerlaw`).
Can you please do it in this PR or do you want me to do this in a separate PR?